react-webpack/webpack.config.js

```diff
@@ -0,0 +1,21 @@
+const path = require('path');
+ 
```

Comment:
Maybe provide some level of explanation what's going on below...

Reply:
Done.